# Kertaus

In [147]:
using Printf
using Dates
@printf "%s\n" Dates.now();
versioninfo(verbose=false);

2021-10-26T20:15:31.816
Julia Version 1.5.4
Commit 69fcb5745b (2021-03-11 19:13 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
Environment:
  JULIA_DEPOT_PATH = C:\Users\snikula\.julia;C:\Users\snikula\AppData\Local\JuliaPro-1.5.4-1\Julia-1.5.4\local\share\julia;C:\Users\snikula\AppData\Local\JuliaPro-1.5.4-1\Julia-1.5.4\share\julia
  JULIA_LOAD_PATH = C:\Users\snikula\.julia\environments\JuliaPro_v1.5.4-1;@;@v#.#;@stdlib
  JULIA_PKG_SERVER = pkg.juliahub.com


k-2 

Kirjan vastaus: a) 16 lx b) 0.57 m

In [148]:
L=10e-3
C=2e-9
f=1/(2*pi*sqrt(L*C))
@printf("Värähtelytaajuus on %.1e Hz.\n", f)


Värähtelytaajuus on 3.6e+04 Hz.


k-3 

Kirjan vastaus: a) 1500 lx b) 630 lx c) voi

In [149]:
L=15e-3
f=97.7e6
C=1/((2*pi*f)^2*L)
@printf("Kapasiteetin tulee olla %.1e F.\n", C)


Kapasiteetin tulee olla 1.8e-16 F.


k-5

Kirjan vastaus: 300 nm - 530 nm

In [150]:
c=2.99792e8
L=1
l=2*L
f=c/l
@printf("Taajuus on %.2e Hz.\n", f)
s=1000e3
t=s/c
@printf("Aika on %.2e s.\n", t)

Taajuus on 1.50e+08 Hz.
Aika on 3.34e-03 s.


k-6

Kirjan vastaus: a) 49° b) 35°

In [151]:
L=0.8e-3
C=2e-12
f=1/(2*pi*sqrt(L*C))
l=c/f
@printf("Aallonpituus kapasiteetilla %.2e F on %.0f m.\n", C,l)
C=50e-12
f=1/(2*pi*sqrt(L*C))
l=c/f
@printf("Aallonpituus kapasiteetilla %.2e F on %.0f m.\n", C,l)

Aallonpituus kapasiteetilla 2.00e-12 F on 75 m.
Aallonpituus kapasiteetilla 5.00e-11 F on 377 m.


k-7

Kirjan vastaus: a) 29.7° b) 46.0°, 43.6°


In [146]:
f=900e6
l=c/f
L=l/2
@printf("Puoliaaltoantennin pituus taajuudella %.2e Hz on %.1e m.\n", f,L)
f=2100e6
l=c/f
L=l/2
@printf("Puoliaaltoantennin pituus taajuudella %.2e Hz on %.1e m.\n", f,L)


Puoliaaltoantennin pituus taajuudella 9.00e+08 Hz on 1.7e-01 m.
Puoliaaltoantennin pituus taajuudella 2.10e+09 Hz on 7.1e-02 m.


k-9

Kirjan vastaus: 0.302 mm punaisen valon osumakohdan alapuolella


k-11

Kirjan vastaus: 656 nm

k-12

Kirjan vastaus: 657 nm, 4.56e14Hz

k-13

Kirjan vastaus: 840 nm

k-16

Kirjan vastaus: a 43 fN

k-17

Kirjan vastaus: 3.2 fN

k-18

Kirjan vastaus: 93 ns

k-19

Kirjan vastaus: b) 0.32 T ja 3.3 MHz

k-20

Kirjan vastaus: b) 64 kV

k-21

Kirjan vastaus: 46 kN/C

k-22

Kirjan vastaus: b) 32 µN itään

k-23

Kirjan vastaus: a) 0.14 A

k-24

Kirjan vastaus: a) 0.31 T b) 58 mN

k-27

Kirjan vastaus: 0.13 V

k-28

Kirjan vastaus: a) 4.2 m/s² oikealle b) 34 m/s

k-29

Kirjan vastaus: c) 1.5 V

k-35

Kirjan vastaus: b) 280 V, 100 Hz

k-36

Kirjan vastaus: a) 2 b) 1

k-37

Kirjan vastaus: a) 20 V, 14 V c) -7.8 V

k-38

Kirjan vastaus: b) 0.9 kΩ c) 6.8 V

k-39

Kirjan vastaus: 260 kierrosta

k-40

Kirjan vastaus: a) kaksinkertaistuu b) pienenee puoleen c) kaksinkertaistuu d) ei muutu

k-41

Kirjan vastaus: a) 1.2 V b) sulaa ja katkeaa

k-42

Kirjan vastaus: 0.73

k-43

Kirjan vastaus: b) 1.57 m

k-44

Kirjan vastaus: 98 µH

S2007/4 b) 38.4°

K2014/+13 d) 1) 99 % 2) 52 %

K2016/8 a) 0.7 mV